app/views/pages/medicaid/amounts_other_income.erb

```diff
@@ -0,0 +1,29 @@
+<div class="step-header">
+  <a href="/pages/medicaid/amounts_self_employment" class="step-header__back-link">
+    <i class="button__icon icon-arrow_back"></i>
+  </a>
+  <h4 class="step-header__title">
+    Income & Expense Amounts
+  </h4>
+</div>
+
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="form-card__title">Tell us more about the specifics of your “other” income.</div>
+    <p class="text--help text--centered">
+      If you don't know the exact number, that's okay. These questions aren't required. However, your income determines the benefits you are eligible for.
```

Comment:
random Q: how much do you care about curly vs straight quotes?

Reply:
I noticed that you're using smart quotes in some places and not others, yes. Not important to me but i know that some care about this